tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -0,0 +1,350 @@
+        warmup_duration = 1  # minutes
+        test_duration = 5  # minutes
+
+        workload = {
+            "name": "MaxConnectionsTest",
+            "topics": 1,
+            "partitions_per_topic": self._optimal_partition_count(),
+            "subscriptions_per_topic": subscriptions,
+            "consumer_per_subscription": consumers,
+            "producers_per_topic": producers,
+            "producer_rate": (tier_config.ingress_rate / 2) // (4 * KiB),
+            "message_size": 4 * KiB,
+            "payload_file": "payload/payload-4Kb.data",
+            "consumer_backlog_size_GB": 0,
+            "test_duration_minutes": test_duration,
+            "warmup_duration_minutes": warmup_duration,
+        }
+
+        # ProducerSwarm parameters
+        #
+
+        producer_kwargs = {}
+        producer_kwargs['min_record_size'] = 64
+        producer_kwargs['max_record_size'] = 64
+
+        effective_msg_size = producer_kwargs['min_record_size'] + (
+            producer_kwargs['max_record_size'] -
+            producer_kwargs['min_record_size']) // 2
+
+        conn_limit = tier_config.connections_limit - 2 * (
+            producers + consumers * subscriptions)
+        _target_per_node = conn_limit // SWARM_WORKERS
+        _conn_per_node = int(_target_per_node * 1)
+
+        msg_rate_per_node = (1 * MiB) // effective_msg_size
+        messages_per_sec_per_producer = max(
+            msg_rate_per_node // _conn_per_node, 1)
+
+        # single producer runtime
+        # Roughly every 500 connection needs 30 seconds to ramp up
+        target_runtime_s = max(30 * math.ceil(_target_per_node // 500),
+                               60 * (test_duration + warmup_duration))
+        records_per_producer = messages_per_sec_per_producer * target_runtime_s
+
+        self._ctx.logger.warn(
+            f"Producers per node: {_conn_per_node} Messages per producer: {records_per_producer} Message rate: {messages_per_sec_per_producer} msg/s"
+        )
+
+        producer_kwargs[
+            'messages_per_second_per_producer'] = messages_per_sec_per_producer
+
+        benchmark = OpenMessagingBenchmark(
+            self._ctx,
+            self.redpanda,
+            "ACK_ALL_GROUP_LINGER_1MS_IDEM_MAX_IN_FLIGHT",
+            (workload, OMBSampleConfigurations.UNIT_TEST_LATENCY_VALIDATOR),
+            num_workers=OMB_WORKERS,
+            topology="ensemble")
+        benchmark.start()
+
+        # Create topic for swarm workers after OMB to avoid the reset
+        swarm_topic_name = "swarm_topic"
+        try:
+            self.rpk.delete_topic(swarm_topic_name)
+        except:
+            pass
+        finally:
+            self.rpk.create_topic(swarm_topic_name,
+                                  tier_config.partitions_upper_limit -
+                                  self._optimal_partition_count(),
+                                  replicas=3)
+
+        swarm = []
+        for _ in range(0, SWARM_WORKERS):
```

Comment:
could be just `range(SWARM_WORKERS)`

Reply:
Switched to what you recommended.

Reply:
Thank, this was definitely a "nit:" but I forgot to note that.